### PR TITLE
skip all ATS tests since OpenID is getting retired

### PR DESCRIPTION
**Pull request by valeriupredoi**
[CI Tests](https://github.com/ESGF/esgf-pyclient/actions/runs/9789072623/job/27028151517) started failing again because the OpenID service at DKRZ has been retired.

### CODE REVIEW

**Comment by valeriupredoi** [comment]
OK yet another barf since there is no more CMIP5 on DKRZ arghhh

**Comment by valeriupredoi** [comment]
OK so the 500-503s are transitory - why them ESGF nodes so flaky at connecting to all of a sudden? I don't remember them being so fragile? @bouweandela what do you say we got us rid of Pythons 3.8 and 3.9, at least only in the CI?

**Comment by bouweandela** [comment]
The OpenID servers that used to be provided by ESGF will stop operating and a new authentication mechanism will be set up in the future. How about unconditionally skipping all of the tests in tests/test_ats.py for now? We can see about removing support for authentication completely from the package in the future. Currently, it is not necessary to log in to access any data on ESGF anyway.

**Comment by valeriupredoi** [comment]
> The OpenID servers that used to be provided by ESGF will stop operating and a new authentication mechanism will be set up in the future. How about unconditionally skipping all of the tests in tests/test_ats.py for now? We can see about removing support for authentication completely from the package in the future. Currently, it is not necessary to log in to access any data on ESGF anyway.

cheers, bud! My thought exactly when I started reading on OpenID changes - lemme do that here :+1:

File: tests/test_context.py [hunk 1]
```diff
@@ -19,8 +19,9 @@ class TestContext(TestCase):
     _test_few_facets = 'project,model,index_node,data_node'
 
     def setUp(self):
-        self.test_service = 'http://esgf-data.dkrz.de/esg-search'
-        # self.test_service = 'https://esgf.ceda.ac.uk/esg-search'
+        # DKRZ have retired CMIP5 data
```
Comment (bouweandela) [inline tests/test_context.py]: Maybe they have removed data from this particular model? There is definitely still CMIP5 data there: https://esgf-data.dkrz.de/esg-search/search?format=application%2Fsolr%2Bjson&project=CMIP5 returns many results.

Comment (valeriupredoi) [reply tests/test_context.py]: eugh maybe they'll remove data from the model we replace it with, in CEDA we trust, man :grin:

**Comment by valeriupredoi** [comment]
@bouweandela shall we get this in, buds? :beer:

File: tests/test_context.py (hunk 1 above)
Comment (bouweandela) [reply tests/test_context.py]:
```suggestion
```
Switching to CEDA for the tests seems fine, but please remove the comment as it is misleading (at least for now)

**Comment by bouweandela** [comment]
Yes. Could you please update the pull request description so it reflects what is in the pull request?

**Comment by valeriupredoi** [comment]
ah true that - done, good point!

**Pull request merged**